site/en/docs/privacy-sandbox/feedback/report-2023-q1.md

```diff
@@ -330,15 +330,15 @@ requests</a>.</td>
       <td style="vertical-align: top;">Clarification on timelines for testing and implementation for FLEDGE
 features such as Fenced Frame enforcement, K-Anonymity, and so forth.</td>
       <td style="vertical-align: top;">We have <a
-href="https://developer.chrome.com/docs/privacy-sandbox/fledge-api/feature-status/">shared a blog post</a> on various scoped FLEDGE features and when they will
```

Comment:
Leave the feedback reports as is. I want them to always reflect the information as of the publication date, rather than getting retroactively updated.